src/model/TaskManager.h

```diff
@@ -33,10 +33,21 @@ class TaskManager : public ModelInterface {
     ActionResult Delete(const Core::TaskID &id, bool deleteChildren) override;
     ActionResult Validate(const Core::TaskID &id) const override;
     ActionResult AddLabel(const Core::TaskID &, const std::string &) override;
+    ActionResult ClearLabel(const Core::TaskID &, const std::string &) override;
```

Comment:
this branch should not me from `main`. you have these changes already in other review